src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms;
```

Comment:
variable could be created for the transition-duration as well

Reply:
created variables for header and card transitions